benchmarks/src/helpers/sim_fallback.jl

```diff
@@ -0,0 +1,24 @@
+# TODO: Do we have anything better solution than a runtime interface?
```

Comment:
I think making a struct for SimulationConfiguration that people can attach functions to would be better than relying on them to overload these default methods.